src/main/java/org/jitsi/videobridge/rest/DebugRequestHandler.java

```diff
@@ -0,0 +1,145 @@
+package org.jitsi.videobridge.rest;
+
+import org.jitsi.osgi.*;
+import org.jitsi.utils.logging.*;
+import org.jitsi.videobridge.*;
+import org.json.simple.*;
+import org.osgi.framework.*;
+
+import javax.servlet.http.*;
+import java.io.*;
+
+import static org.jitsi.videobridge.rest.HandlerImpl.DEBUG;
+
+/**
+ * Handles requests for "/colibri/debug*".
+ *
+ * Note that using this interface MAY disrupt running conferences or even
+ * cause a deadlock. It is really meant only for debugging, which is why it is
```

Comment:
The conference interruption I get but potentially causing a deadlock seems bad. Can it be avoided?

Reply:
I've tried to avoid acquiring locks, or holding any objects (I just push primitives and JSON/NodeStatsBlock objects), but it traverses a big chunk of the classes down from Conference and it's hard to guarantee. I don't have a specific deadlock in mind.

Reply:
Ah ok, I thought you meant you were seeing one occur.